### ISSUE TITLE
How to stop logging and view error report before sending

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-app-android#3175]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
https://www.coronawarn.app/en/faq/#error_log and
https://www.coronawarn.app/en/blog/2021-05-12-corona-warn-app-version-2-2/
do not answer the question.
* [ ] Question is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Question has not already been asked in another issue

## Your Question

The question is how to view exactly the data which the Error Report sends to technical support. There seems to be no way to stop error logging, view the data, then send it.

Also I could find no way, after the data has been sent, to view what data was sent. The only remnant on the mobile device seems to be the timestamp and the ID.

App version: 2.2.1

In App Information > Error Reports, the text (from string `debug_log_privacy_card_container_section_two`) says:

"Initially, only you will have access to the error report. You can then decide whether to send the error report to the technical support team, whether to save the error report on your smartphone first, or whether to stop and erase the recorded information. **If you first save the error report locally, this will give you the opportunity to view the error report yourself before submitting it to support.**"

![image](https://user-images.githubusercontent.com/66998419/118269316-e1340200-b4be-11eb-9d6e-206d04e89e65.png)

After tapping "START" there is no option to STOP and VIEW the data. 
- The "STOP AND DELETE" button does what it says, so there is no data to view.
- The "SHARE AND CONTINUE" button opens up the Android Share option (by default with apps Drive, Bluetooth and Gmail available) however "Recording Active" is still shown, so it is still collecting data as can also be seen by the fact that the Current size continues to increase.
- The "SEND ERROR REPORT" only provides the option to "AGREE AND SEND".

![image](https://user-images.githubusercontent.com/66998419/118269920-95358d00-b4bf-11eb-90ec-b7ca96cd8265.png)

**Comment by dsarkar** [comment]
@MikeMcC399 Press share and continue .... hm, not very intuitive?

**Comment by MikeMcC399** [comment]
@dsarkar 
I'm not sure whether this is a wishlist item or a bug, if you say there is no possibility to do what the text tells you to do.

Since you say this affects both Android and iOS, please move to documentation. When you get some feedback I would either submit a wishlist item or perhaps modify this submission.

Also I agree with your comments (in the previous edit) where you said the share and continue was not intuitive. I didn't know what "share" meant (with whom? - I thought maybe share with support?) or what "continue" meant (continue logging? continue sharing? continue using?). This point could be a wishlist item to label the button better or describe the action better.

**Comment by dsarkar** [comment]
@MikeMcC399 

Agreed, the text does not describe strictly correct the process: The text sounds a bit like: STOP - (optionally) VIEW - (Optionally) SEND. 

Several options to fix this issue:

1. Adapt the text to the "reality" and maybe explain the SHARE button better and/or rename the SHARE button. (More or less your last sentence)
2. Or change the work flow: 
  - Introduce separate functions: START, STOP, CONTINUE, DELETE error report logging
  - Introduce separate functions: SEND to support, SHARE/VIEW error report
